### PR TITLE
Prometheus Metrics

### PR DESCRIPTION
**Pull request by tanneberger**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -69,6 +70,15 @@ pub fn create_db_pool() -> DbPool {
     Pool::new(manager).expect("Failed to create pool.")
 }
 
+pub fn get_prometheus() -> PrometheusMetrics {
+    PrometheusMetricsBuilder::new("api")
+        .endpoint("/metrics")
+        //.const_labels(None)
+        .build()
+        .unwrap()
+
```
Comment (gshipunov) [inline src/main.rs]:
```suggestion
```
make linter happy

**Review by gshipunov — CHANGES_REQUESTED** [review]
Looks good, but need to make rust-fmt happy so clippy runs

**Comment by tanneberger** [comment]
want to checkout how this works and then if it works nicely add it to all our services using actix

**Review by gshipunov — COMMENTED** [review]
goffik af

**Pull request merged**